standards/request-response.md

```diff
@@ -471,6 +471,28 @@ X-SPS-User: 123456789012345678901234567890
 The usage of non-standard headers is not considered custom headers. For example, your API may consume or interact with other infrastructure outside of the contract specification for your service, such as `X-Forwarded-Host` or `X-Request-ID`, these are appropriate to continue using, but would not expect to identify them in your Open API spec for example. As such these types of headers not relevant to REST are intentionally left out of this documentation.
 ```
 
+#### SPS-CORS-Error
+
+**Type**: Response
+
+**Support**: OPTIONAL
+
+**Description**: A [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) error with an invalid request on a non-200 status code can be difficult to understand the reasons why the request was rejected. Using a custom header provides the ability to include more detailed information to the consumer on why their CORS preflight OPTIONS request was rejected.
+
+- The header **MUST** be added when a `CORS` `OPTIONS` request is rejected.
+- A response with this header **SHOULD** indicate a status code of `403 - Forbidden`.
+- The header **MUST** contain one of the following values:
+    - __bad origin__ - Indicates to the consumer that the provided request header `Origin` header did not match approved list of source URLs.
```

Comment:
```suggestion
    - __bad origin__ - Indicates to the consumer that the provided request header `Origin` did not match approved list of source URLs.
```

Reply:
Might be overly pedantic on my part, but I wonder if the words `domains` or `hostnames` might be a better choice than `URLs` here. URL makes me think that it can (and should) be constrained to a resource with a directory path.

Reply:
Love it, I'll hostnames to be consistent with terminology in URL Structure here https://spscommerce.github.io/sps-api-standards/standards/url-structure.html#overview